### ISSUE TITLE
Compromise doesn't work with NextJS 12

### ISSUE THREAD

**Issue by baptistemanson** [spencermountain/compromise#904]
Currently, Compromise uses 
```js
let env = typeof process === 'undefined' ? self.env || {} : process.env
if (env.DEBUG_TAGS) {
    
}
```
With the webpack DefinePlugin, which is now used in NextJS 12, process is an empty object {}. It cannot be changed, for security purposes.
It results in a runtime FATAL error (cannot read DEBUGS_TAGS of undefined).
It could be fixed by replacing by:
```js
let env = typeof process === 'undefined' || !process.env ? self.env || {} : process.env
if (env.DEBUG_TAGS) {
    
}
```
which is more correct, as it doesnt assume that env must be present in process.

**Comment by spencermountain** [comment]
whoa!
i will change this today, and add it in the release next week.
thank you!

**Comment by spencermountain** [comment]
fixed in 14.1.0
cheers!
